views/chitter/index.erb

```diff
@@ -0,0 +1,23 @@
+<%if @user%>
+<h2>Hey, <%=@user.username%>!</h2>
+<br>
+<form action='/chitter/peep' method='post'>
+  <input type='text' name='message' placeholder='What&#39;s on your mind?'
+    maxlength="512" required><br>
+  <input type='text' name='tag' placeholder='Tag a friend with their username!'
```

Comment:
I don't think tagging a friend was part of the objectives, but i really love this extra attention to detail!